### ISSUE TITLE
[Request] Add self-contained powers (as on consumables) to equipment items

### ISSUE THREAD

**Issue by FoxLee** [EndlesNights/dnd4eBeta#173]
I don't know if this is possible, as I'm not aware how similar these two item types really are under the hood. But I'm sure I won't be the only person to wonder about this, so filing an issue seemed to be a good idea for future reference.

Is feasible to add self-contained powers to equipment items? So far I have been making the power separately and just adding both to the actor's sheet, and only recently did I realise that the consumable item type has a power built-in by design. If the same can be done for gear, that would be really handy for modelling 4e equipment.

Granted, some items have more than one power, and I assume it's not practical to include an arbitrary number of self-contained powers on anything  but an actor. But even having one self-contained power would be enough for _most_ popular equipment.

**Comment by draconas1** [comment]
This one is on me, and is not done for complexity.  Simple put its a PITA on a sheet that is already very complicated.  Hence instead I added Item as a power type so people could make their item powers.  

Consumables use a stripped down action sheet that do not fully implement power rules or features and that is causing enough headaches, see your other issue.  I decided to try for consumables because rulebook (aka non dragon magazine) consumables seemed to have a fairly stripped down power set and ways to interact and turns out I still missed some stuff.  Item powers run a huge range, so it would basically be a complete reimplementation of powers, skipping the resource usage and weapon usage.  

Basically it was a lot more headache than I wanted (especially compared to adding 1 extra power type to a dropdown, the coloured CSS was already waiting for me), and I am modelling them by having a separate power with a note in its description of where it comes from.

**Comment by FoxLee** [comment]
Hmm, so doable but probably too much work to happen any time soon, if it does happen at all? That's very reasonable. I'd like to work on it myself, but it's _definitely_ over my head until I get a better handle on the code.

"Item" as a usage type doesn't really cover non-consumable item powers though, since they all have a normal usage attached, which info one wouldn't want to lose. (I didn't even realise it was there—I've been putting "Item" in the level field since it can handle text.) I might add it as a power type option if nobody has an objection to that.

**Comment by FoxLee** [comment]
Closing this as it now falls under #343
